README.md

```diff
@@ -24,6 +24,14 @@ git clone https://github.com/HuidaeCho/grass-conda.git
 cd grass-conda
 conda-build recipe
 ```
+
+If it is on WSL, there may have issue with spaces in PATH environment variable, we can get rid of them before building with 
+
+```bash
+PATH="$CONDA_PREFIX/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
I still don't understand why this should be part of this recipe. Please show me how your build breaks without this.